db/migrate/20220609221942_remove_orphaned_owners.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveOrphanedOwners < ActiveRecord::Migration[7.0]
+  def up
+    Ownership.includes(:user).where(user: {id: nil}).destroy_all
```

Comment:
:information_source: This can be written today as `Ownership.where.missing(:user).destroy_all`.

Reply:
Oh nice, updated!